ripley/externals/canvas.py

```diff
@@ -47,6 +47,34 @@ def ping_canvas():
     return get_account(app.config['CANVAS_BERKELEY_ACCOUNT_ID']) is not None
 
 
+def create_external_tool(account_id, client_id):
+    account = get_account(account_id, api_call=False)
+    tool = None
+    try:
+        tool = account.create_external_tool(
+            client_id=client_id,
+            name='name',
+            privacy_level='public',
+            consumer_key='consumer_key',
+            shared_secret='shared_secret',
```

Comment:
@lyttam - Are the values above placeholders (eg, `consumer_key`) or simply not used?